package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "smart-starter-app",
+  "version": "1.0.0",
+  "author": "Cerner Corporation, Cloud App Dev Platform",
+  "description": "== README",
+  "main": "./src/starter_app.js",
+  "style": "dist/css/starter_app.css",
+  "less": "src/less/manifest.less",
+  "license": "== LICENSE",
+  "homepage": "https://github.com/parthivbhagat/pb026393.github.io",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:parthivbhagat/pb026393.github.io.git"
+  },
```

Comment:
See if you can use this npm package dependency for fhirclient instead of pulling the js file from their repo.

"dependencies": {
    "fhirclient": "^0.1.9"
  }

https://www.npmjs.com/package/fhirclient